### PR TITLE
tools/dm: add DM-worker intro

### PR DESCRIPTION
**Pull request by CaitinChen**
Related PR: https://github.com/pingcap/docs/pull/915
English file:
https://github.com/pingcap/docs/blob/master/tools/dm/dm-worker-intro.md

@lilin90 @GregoryIan  @csuzhangxc PTAL

### CODE REVIEW

File: tools/dm/dm-worker-intro.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+## DM-worker 所需权限
+
+本小节主要介绍使用 DM-worker 时所需的上下游数据库用户权限以及各处理单元所需的用户权限。
+
+### 上游数据库用户权限
+
+上游数据库 (MySQL/MariaDB) 用户必须拥有以下权限：
+
+| 权限 | 作用域 |
+|----|----|
```
Comment (lilin90) [inline tools/dm/dm-worker-intro.md]:
```suggestion
|:----|:----|
```

File: tools/dm/dm-worker-intro.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+```
+
+如果还要同步其他数据库的数据到 TiDB, 请确保已赋予这些库跟 `db1` 一样的权限。
+
+### 下游数据库用户权限
+
+下游数据库 (TiDB) 用户必须拥有以下权限：
+
+| 权限 | 作用域 |
+|----|----|
```
Comment (lilin90) [inline tools/dm/dm-worker-intro.md]:
```suggestion
|:----|:----|
```

File: tools/dm/dm-worker-intro.md [hunk 3]
```diff
@@ -0,0 +1,93 @@
...
+对要执行同步操作的数据库或表执行下面的 `GRANT` 语句：
+
+```sql
+GRANT SELECT,INSERT,UPDATE,DELETE,CREATE,DROP,ALTER,INDEX  ON db.table TO 'your_user'@'your_wildcard_of_host';
+```
+
+### 处理单元所需的最小权限
+
+| 处理单元 | 最小上游 (MySQL/MariaDB) 权限 | 最小下游 (TiDB) 权限 | 最小系统权限 |
+|----|--------------------|------------|----|
```
Comment (lilin90) [inline tools/dm/dm-worker-intro.md]:
```suggestion
|:----|:--------------------|:------------|:----|
```

**Comment by CaitinChen** [comment]
@lilin90 Thanks. PTAL again.

File: tools/dm/dm-worker-intro.md [hunk 4]
```diff
@@ -0,0 +1,93 @@
...
+### 处理单元所需的最小权限
+
+| 处理单元 | 最小上游 (MySQL/MariaDB) 权限 | 最小下游 (TiDB) 权限 | 最小系统权限 |
+|:----|:--------------------|:------------|:----|
+| Relay log | `REPLICATION SLAVE` (读取 binlog）<br>`REPLICATION CLIENT` (`show master status`, `show slave status`) | 无 | 本地读/写磁盘 |
+| Dumper | `SELECT`<br>`RELOAD`（获取读锁将表数据刷到磁盘，进行一些操作后，再释放读锁对表进行解锁）| 无 | 本地写磁盘 |
+| Loader | 无 | `SELECT`（查询 checkpoint 历史）<br>`CREATE`（创建数据库或表）<br>`DELETE`（删除 checkpoint）<br>`INSERT`（插入 dump 数据）| 读/写本地文件 |
+| Binlog replication | `REPLICATION SLAVE`（读 binlog）<br>`REPLICATION CLIENT` (`show master status`, `show slave status`) | `SELECT`（显示索引和列）<br>`INSERT` (DML)<br>`UPDATE` (DML)<br>`DELETE` (DML)<br>`CREATE`（创建数据库或表）<br>`DROP`（删除数据库或表）<br>`ALTER`（修改表）<br>`INDEX`（创建或删除索引）| 本地读/写磁盘 |
+
+> **注意:** 这些权限并非一成不变。随着请求改变，这些权限也会改变。
```
Comment (csuzhangxc) [inline tools/dm/dm-worker-intro.md]:
```suggestion
> **注意:** 这些权限并非一成不变。随着需求改变，这些权限也可能会改变。
```

**Pull request merged**